admissions/urls.py

```diff
@@ -8,4 +8,6 @@
 urlpatterns = [
     path("", IndexView.as_view(), name="index"),
     path("detail/<int:application_id>", views.detail, name="detail"),
+    path("accept/<int:application_id>", views.accept, name="accept"),
+    path("reject/<int:application_id>", views.reject, name="reject"),
```

Comment:
i think you can combine this into one.

Reply:
Please provide some context into this before resolving this issue.

Reply:
> Please provide some context into this before resolving this issue.

like I said, this should remain no change. Since these two are functions been called. And I already passed ID as parameter, then passing another flag(0/1/2) will mess the url